src/main/java/com/google/data/Authentication.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
```

Comment:
I think we have to add the Apache License on the top? Not entirely sure.

Reply:
@ShreeragJ Do we?

Reply:
I'm not sure, Sergio where did you see that?

Reply:
I don't remember exactly but I think it was a meeting about open source projects. I'm not entirely sure to be honest. Also, the java files in the example repository have the license as a comment on the top

Reply:
Probably safe to add them in that case.

Reply:
Sounds good, I'll add it in